x-pack/plugin/stack/build.gradle

```diff
@@ -0,0 +1,29 @@
+evaluationDependsOn(xpackModule('core'))
+
+apply plugin: 'elasticsearch.esplugin'
+
+esplugin {
+  name 'x-pack-stack'
+  description 'Elasticsearch Expanded Pack Plugin - Stack'
+  classname 'org.elasticsearch.xpack.stack.StackPlugin'
+  extendedPlugins = ['x-pack-core']
+  hasNativeController false
+  requiresKeystore true
+}
+archivesBaseName = 'x-pack-stack'
+
+dependencies {
+  compileOnly project(path: xpackModule('core'), configuration: 'default')
+  testImplementation project(path: xpackModule('core'), configuration: 'testArtifacts')
+}
+
+// add all sub-projects of the qa sub-project
+// gradle.projectsEvaluated {
+//   project.subprojects
+//     .find { it.path == project.path + ":qa" }
+//     .subprojects
+//     .findAll { it.path.startsWith(project.path + ":qa") }
+//     .each { check.dependsOn it.check }
+// }
```

Comment:
do we need to enable this before merging?

Reply:
Yep, I'll enable it